packages/blockchain-wallet-v4-frontend/src/scenes/Login/index.js

```diff
@@ -16,6 +16,8 @@ class LoginContainer extends React.Component {
   onSubmit (event) {
     event.preventDefault()
     const { guid, password, code } = this.props
+    localStorage.setItem('ls.guid', JSON.stringify(guid))
```

Comment:
I think we're doing this a bit too soon here, and should instead save it after a successful login attempt. We should also save the session token. See: https://github.com/blockchain/My-Wallet-V3-Frontend/blob/4e892c2363b4bd3b3c4d8c7aed053b7586bc4b93/assets/js/services/wallet.service.js#L172-L181